debian/debhelper-build-stamp

```diff
@@ -0,0 +1 @@
+install-win31
```

Comment:
This is auto-generated I think.
Please do `git clean -fd`

Reply:
Whoops I really didn't pay proper attention with this one. Should be better now!